gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/spi/DefaultPubSubRpc.java

```diff
@@ -117,7 +151,7 @@ public V apply(ApiException exception) {
 
   @Override
   public Future<Topic> create(Topic topic) {
-    // TODO: it would be nice if we can get the idempotent inforamtion from the ApiCallSettings
+    // TODO: it would be nice if we can get the idempotent information from the ApiCallSettings
```

Comment:
Guava is used internally (see imports for this file) but not exposed in the API.
Are you asking because you would have expected it to return 'ListenableFuture'?